src/Microsoft.Extensions.Logging.Log4Net.AspNetCore/Log4NetExtensions.cs

```diff
@@ -17,21 +17,15 @@ public static class Log4NetExtensions
         /// </summary>
         /// <param name="factory">The factory.</param>
         /// <param name="log4NetConfigFile">The log4net Config File.</param>
+        /// <param name="logFile">The file name of the Log File.</param>
+        /// <param name="watch">If this flag is specified and set to true then the framework will watch the configuration file and will reload the config each time the file is modified.</param>
         /// <returns>The <see cref="ILoggerFactory"/>.</returns>
-        public static ILoggerFactory AddLog4Net(this ILoggerFactory factory, string log4NetConfigFile)
+        public static ILoggerFactory AddLog4Net(this ILoggerFactory factory,
```

Comment:
I'll prefer to add a new extension method instead modifing an existing one, as could be a breaking change for any project that currently are using this method.

Here we should just modify the original extension method, using the defaults that you set up as optional parameters, and create a new one to receive concrete parameter values.

Reply:
And what about !NETCOREAPP1_1 section?